src/components/Stats/index.module.css

```diff
@@ -5,14 +5,23 @@
   align-items: center;
   justify-content: space-between;
 
-  padding: $size-26 0;
+  padding: $size-16 0;
 
   border-bottom: 1px solid $color-blue;
+
+  mixin Breakpoint-tabletOnly {
```

Comment:
I don't think this works :p and I think plain media queries with variables is better and easier to understand and tweak as needed